### PR TITLE
[ATIP] Update README.md

### PR DESCRIPTION
**Pull request by Honry**
BUG=https://crosswalk-project.org/jira/browse/XWALK-4682

### CODE REVIEW

File: tools/atip/README.md [hunk 1]
```diff
@@ -216,15 +216,167 @@ def i_visit_url(context, url):
 	</tr>	
 </table>
 
+* Uiautomator Library Overview
+
+<table>
```
Comment (zqzhang) [inline tools/atip/README.md]: Indent in space rather than tab.

Comment (zqzhang) [inline tools/atip/README.md]: Indent in space rather than tab.

File: tools/atip/README.md [hunk 2]
```diff
@@ -216,15 +216,167 @@ def i_visit_url(context, url):
...
+	</tr>
+	<tr>
+		<td></td>
+		<td>Get object info by text</td>
+	</tr>	
+	<tr>
+		<td></td>
+		<td>Save object info to temp</td>
+	</tr>	
+</table>
```
Comment (zqzhang) [inline tools/atip/README.md]: These are code snippets rather than text.

Comment (Honry) [inline tools/atip/README.md]: This is align with WebDriver lib to use text desc for more readable.

File: tools/atip/README.md (hunk 1 above)
Comment (Honry) [inline tools/atip/README.md]: Thanks, I will update it.

**Pull request merged**